packages/lib/modules/pool/PoolList/PoolListFilters.tsx

```diff
@@ -380,12 +385,71 @@ const FilterButton = forwardRef<ButtonProps, 'button'>((props, ref) => {
   )
 })
 
+function ProtocolVersionFilter() {
+  const {
+    togglePoolType,
+    setProtocolVersion,
+    protocolVersion,
+    poolTypes,
+    activeProtocolVersionTab,
+    setActiveProtocolVersionTab,
+  } = usePoolListQueryState()
+
+  const tabs =
+    isDev || isStaging
+      ? PROTOCOL_VERSION_TABS
+      : PROTOCOL_VERSION_TABS.filter(tab => tab.value !== 'v3')
+
+  function toggleTab(option: ButtonGroupOption) {
+    setActiveProtocolVersionTab(option)
+  }
+
+  useEffect(() => {
+    if (protocolVersion === 3) {
+      setActiveProtocolVersionTab(tabs[2])
+    } else if (protocolVersion === 2) {
+      setActiveProtocolVersionTab(tabs[1])
+    } else if (poolTypes.includes(GqlPoolType.CowAmm)) {
+      setActiveProtocolVersionTab(tabs[3])
+    } else {
+      setActiveProtocolVersionTab(tabs[0])
+    }
+  }, [])
+
+  useEffect(() => {
+    if (activeProtocolVersionTab.value === 'cow') {
+      togglePoolType(true, GqlPoolType.CowAmm)
```

Comment:
Choosing CowAmm sets pool type but choosing other options should undo this selection.

Reply:
ahh good catch, I had something in for that initially but then it was causing other issues. Will fix.